### PR TITLE
mode debug to Admin Task

### PR DESCRIPTION
**Pull request by allamand**
Signed-off-by: Sebastien Allamand <sallaman@amazon.com>

Fixes # remove Debug Task, and configure it at an Admin bootstrap task.

### CODE REVIEW

File: .projenrc.js [hunk 1]
```diff
@@ -42,18 +42,12 @@ const project = new AwsCdkTypeScriptApp({
...
     //vpc_tag_name: 'ecsworkshop-base/BaseVPC', // TAG Name of the VPC to create the cluster into (or 'default' or remove to create new one)
     enablePrivateLink: 'false', // this parameter seems to works only one
-    useEFS: false,
-
-    //os_domain: 'magento-cdk4', // default to $CDK_STACK_NAME
-    os_master_user_name: 'magento-master-os',
-
-    //db_name: 'magento3', // default to env $CDK_STACK_NAME
-    db_user: 'magentodbuser',
+    useEFS: 'yes',
```
Comment (Ssriram83) [inline .projenrc.js:45]: Please add comment on what this parameter does

File: docker/Dockerfile.noefs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM public.ecr.aws/seb-demo/magento:latest
```
Comment (Ssriram83) [inline docker/Dockerfile.noefs:1]: TODO - We need to change it to a private repo / bitnami official docker file.

File: docker/entrypoint.sh [hunk 3]
```diff
@@ -27,15 +27,32 @@ if [[ "$1" = "/opt/bitnami/scripts/magento/run.sh" || "$1" = "/opt/bitnami/scrip
...
+
+    info "**update magento credentials"
+    #TODO: do it only on Admin ?
+    cd /bitnami/magento
+    mkdir -p /bitnami/magento/var/composer_home/
+    cat <<END > /bitnami/magento/var/composer_home/auth.json
+{
+    "http-basic": {
+        "repo.magento.com": {
+            "username": "$MAGENTO_MARKETPLACE_PUBLIC_KEY",
```
Comment (Ssriram83) [inline docker/entrypoint.sh:42]: TODO - We need to have an instruction late on populating this as part of the readme

**Review by Ssriram83 — APPROVED** [review]

File: docker/Dockerfile.noefs (hunk 2 above)
Comment (allamand) [reply docker/Dockerfile.noefs:1]: Yes: I'll flag that we need to have this image from a CI automated build pipeline : https://github.com/allamand/cdk-magento-fargate/issues/31

File: docker/entrypoint.sh (hunk 3 above)
Comment (allamand) [reply docker/entrypoint.sh:42]: I added them in the **Project Pre-requisites** section of the README

File: .projenrc.js (hunk 1 above)
Comment (allamand) [reply .projenrc.js:45]: It is done in the readme, you want also comments in this file ?

Comment (Ssriram83) [reply .projenrc.js:45]: This should be ok.

**Review by Ssriram83 — APPROVED** [review]

**Pull request merged**